### DISCUSSION TITLE
Is ESM possible?

### DISCUSSION THREAD

**Discussion by fev4** [nanojsx/nano#43] [General]
I just found the project, and after quickly comparing it against my use-cases, I find that I'm quite dearly missing ES Modules, similar to what Preact has [https://unpkg.com/preact?module](https://unpkg.com/preact?module).

Is ESM possible?

**Comment by yandeu** [comment]
I haven't added it because I use denofiy to convert the code to work on Deno.
When I started denoify, did not support ESM. But now, denoify supports ESM and I could ship NanoJSX as ESM.
